crates/oxc_formatter/src/lib.rs

```diff
@@ -33,6 +53,8 @@ pub struct Formatter {
 
     // Quote property with double quotes
     quote_property_with_double_quotes: bool,
+
+    final_end_of_line: FinalEndOfLine,
```

Comment:
Can we reference all options to the prettier doc, i.e. https://prettier.io/docs/en/options.html#end-of-line for this one.

In oxc, everything has a reference to some documentation or spec if it appears to come out of no where.